src/ray/stats/metric_defs.cc

```diff
@@ -196,6 +185,43 @@ DEFINE_stats(gcs_storage_operation_count,
              (),
              ray::stats::COUNT);
 
+/// Object store
+/// Local object store memory of different types reported from multiple
+/// components. Reported metric from different subtypes, e.g. Location
+/// and Type could overlap. Reported metric within the same type might
+/// also overlap.
+DEFINE_stats(object_store_memory_bytes,
+             "Object store memory by various sub-kinds on this node",
+             /// Location:
+             ///    - InMemory: currently in memory (ObjectManager)
+             ///    TODO(rickyx): current spilled
+             ///    - Spilled: spilled to disk (LocalObjectManager)
+             ///    - Fallback: fallback allocated (ObjectManager)
+             /// Type:
+             ///     reported from LocalObjectManager
+             ///    - PrimaryCopy: pinned by local raylet and pending spills
+             ///    - PendingSpill: still in memory but to be spilled
+             ///    - PendingRestore: to be restored from spill
```

Comment:
PendingRestore shouldn't be reported right? Since that's not actual memory usage.

Reply:
Yeah - they do not correspond to a physical chunk of usage, but more of a logical type.